examples/client.c

```diff
@@ -288,7 +336,12 @@ int main(int argc, char *argv[]) {
 
     struct ev_loop *loop = ev_default_loop(0);
 
-    ev_io_init(&watcher, recv_cb, conn_io->sock, EV_READ);
+#ifdef _WIN32
+    int sockfd = _open_osfhandle(conn_io->sock, 0);
```

Comment:
Seems like there is EV_WIN32_HANDLE_TO_FD(h) macro to do the same.

Reply:
That is a private symbol used in ev.c, it is not exported via ev.h.